repose-aggregator/components/filters/valkyrie-authorization/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -3,12 +3,16 @@ package org.openrepose.filters.valkyrieauthorization
 import java.io.InputStream
```

Comment:
Using IntelliJ's auto-formatted changed a bunch of things in this class. Something to keep in mind.